server/gateway/services/UserService.js

```diff
@@ -0,0 +1,217 @@
+'use strict';
+
+var AppUtil = require('../../libs/AppUtil');
+var Logging = require('../utilities/Logging');
+var config = require('config');
+var _ = require('lodash');
+var PubSub = require('../../libs/PubSub/PubSubAdapter');
+var Message = require('../../libs/PubSub/Message');
+var constants = require('../../Constants');
+var PubSubChannels = require('../../PubSubChannels');
+var async = require('async');
```

Comment:
Isn't "async" a keyword in Node JS? I'm not an expert in Node JS, but if this is legal in Node JS, I think it would still be better to not use keyword as variable name.